step.yml

```diff
@@ -105,30 +82,20 @@ inputs:
   - keystore_password: $BITRISEIO_ANDROID_KEYSTORE_PASSWORD
     opts:
       title: "Keystore password"
-      summary: ""
-      description: ""
       is_required: true
       is_sensitive: true
   - keystore_alias: $BITRISEIO_ANDROID_KEYSTORE_ALIAS
     opts:
-      title: "Keystore alias"
-      summary: ""
-      description: ""
+      title: "Key alias"
```

Comment:
both summary and description are required: https://devcenter.bitrise.io/contributors/create-your-own-step/#the-summary please do not remove but fill them with value.

agree if this is not scope of the task, a follow-up task ticket creation is also acceptable

Reply:
(please do the same with the other occurrences)